packages/start/server/render.ts

```diff
@@ -12,8 +12,8 @@ export function renderSync(
     renderId?: string;
   }
 ) {
-  return () => apiRoutes({
-    forward: inlineServerFunctions({
+  return () => inlineServerFunctions({
+    forward: apiRoutes({
```

Comment:
Using `apiRoutes` and `inlineServerFunctions` middleware again here doesn't make much sense to me right now.  Just looking at it I am expecting these two middlewares to be called twice when requesting a page route...

Reply:
Fixed in 5e615768f87b2270c5c73a7aab802cbaeb33a036